spec/rubocop/cop/rspec/repeated_example_spec.rb

```diff
@@ -89,4 +89,185 @@
       end
     RUBY
   end
+
+  # 新しいテストケース: 複数の重複がある場合
+
+  it 'shows all duplicate line numbers when there are three duplicates' do
+    expect_offense(<<~RUBY)
+      describe 'doing x' do
+        it "does x" do
+        ^^^^^^^^^^^^^^ Don't repeat examples within an example group. Repeated on line(s) 6, 10.
+          expect(foo).to be(bar)
+        end
+
+        it "does y" do
+        ^^^^^^^^^^^^^^ Don't repeat examples within an example group. Repeated on line(s) 2, 10.
+          expect(foo).to be(bar)
+        end
+
+        it "does z" do
+        ^^^^^^^^^^^^^^ Don't repeat examples within an example group. Repeated on line(s) 2, 6.
+          expect(foo).to be(bar)
+        end
+      end
+    RUBY
+  end
+
+  it 'shows all duplicate line numbers when there are four duplicates' do
+    expect_offense(<<~RUBY)
+      describe 'doing x' do
+        it "first" do
+        ^^^^^^^^^^^^^ Don't repeat examples within an example group. Repeated on line(s) 5, 8, 11.
+          expect(foo).to be(bar)
+        end
+        it "second" do
+        ^^^^^^^^^^^^^^ Don't repeat examples within an example group. Repeated on line(s) 2, 8, 11.
+          expect(foo).to be(bar)
+        end
+        it "third" do
+        ^^^^^^^^^^^^^ Don't repeat examples within an example group. Repeated on line(s) 2, 5, 11.
+          expect(foo).to be(bar)
+        end
+        it "fourth" do
+        ^^^^^^^^^^^^^^ Don't repeat examples within an example group. Repeated on line(s) 2, 5, 8.
+          expect(foo).to be(bar)
+        end
+      end
+    RUBY
+  end
+
+  it 'correctly reports duplicates with string interpolation' do
+    expect_offense(<<~RUBY)
+      describe 'doing x' do
+        let(:date) { '2024-06-25' }
+
+        it { expect(subject).to be_urgent(order, now: T("\#{date}T12:00")) }
+        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Don't repeat examples within an example group. Repeated on line(s) 5.
+        it { expect(subject).to be_urgent(order, now: T("\#{date}T12:00")) }
+        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Don't repeat examples within an example group. Repeated on line(s) 4.
+      end
+    RUBY
+  end
+
+  it 'does not flag examples with different string interpolation values' do
+    expect_no_offenses(<<~RUBY)
+      describe 'doing x' do
+        let(:date) { '2024-06-25' }
+
+        it { expect(subject).to be_urgent(order, now: T("\#{date}T12:00")) }
+        it { expect(subject).to be_urgent(order, now: T("\#{date}T17:00")) }
+      end
+    RUBY
+  end
+
+  it 'shows line numbers for real-world scenario from GitHub issue' do
```

Comment:
Isn’t this example covered more or less by `it 'correctly reports duplicates with string interpolation' do` in line 139?